synapse/tests/test_lib_storm.py

```diff
@@ -226,6 +226,39 @@ async def test_lib_storm_basics(self):
 
             self.eq({'foo': 'bar'}, await core.callStorm('return($lib.dict(    foo    =    bar   ))'))
 
+            ddef0 = await core.callStorm('return($lib.dmon.add(${ $lib.queue.gen(hehedmon).put(lolz) $lib.time.sleep(10) }, name=hehedmon))')
```

Comment:
this feels like we're just throwing more and more and more stuff into this particular unit test

Reply:
yep!  i am.  to keep the cortex instantiation count down for cluster tests.